collector.go

```diff
@@ -107,6 +117,10 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 		ch <- prometheus.MustNewConstMetric(c.eventTotal, prometheus.CounterValue, et.count, et.dag, et.task, et.event)
 	}
 
+	for _, st := range m.dagRunStates {
+		ch <- prometheus.MustNewConstMetric(c.dagRunStates, prometheus.GaugeValue, st.count, st.dag, st.state)
```

Comment:
The value cannot go down, right? So shouldn't it be a `CounterValue` instead of a `GaugeValue`?

Reply:
It can go down. The state of a single dag_run changes depending on the results of the latest attempt. We want to alert on `airflow_dag_run_state{state="failed"} > 0`, and then re-running or marking a run as success with clear the alert.

Reply:
Allright :)